PWG/EMCAL/EMCALtasks/AliEmcalCorrectionClusterNonLinearity.cxx

```diff
@@ -39,8 +39,8 @@ AliEmcalCorrectionClusterNonLinearity::AliEmcalCorrectionClusterNonLinearity() :
   fEnergyDistBefore(0),
   fEnergyTimeHistBefore(0),
   fEnergyDistAfter(0),
-  fEnergyTimeHistAfter(0)
-
+  fEnergyTimeHistAfter(0),
+  fSetClusterE(0)
```

Comment:
Please init with `kFALSE` so that we don't have to rely on implicit conversions (not that this would cause a problem here, but best to be consistent)